app/constants/task-type.js

```diff
@@ -3,5 +3,6 @@ module.exports = {
   CREATE_WORKLOAD: 'CREATE-WORKLOAD',
   PROCESS_IMPORT: 'PROCESS-IMPORT',
   PROCESS_REDUCTIONS: 'PROCESS-REDUCTIONS',
-  PROCESS_ADJUSTMENTS: 'PROCESS-ADJUSTMENTS'
+  PROCESS_ADJUSTMENTS: 'PROCESS-ADJUSTMENTS',
+  COURT_REPORTS_CALCULATIONS: 'COURT-REPORTS-CALCULATIONS'
```

Comment:
Some merging will be needed - my branch has `CALCULATION`

Reply:
changed to COURT_REPORTS_CALCULATION